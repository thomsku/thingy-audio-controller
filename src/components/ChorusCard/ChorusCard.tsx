import React from "react";

interface IProps {
  buttonPressed: boolean;
}

const ConnectButton: React.FunctionComponent<IProps> = (props) => {
  return (
    <div className="card">
      <p>press the button</p>
      {props.buttonPressed ? (
        <p>chorus is <span className="on">on</span></p>
      ) : (
        <p>chorus is <span className="off">off</span></p>
      )}
    </div>
  );
};

export default ConnectButton;
