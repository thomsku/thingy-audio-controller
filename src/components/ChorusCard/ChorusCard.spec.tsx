import { shallow, ShallowWrapper } from "enzyme";
import React from "react";
import ChorusCard from "./ChorusCard";

describe("PlatButton", () => {
  let wrapper: ShallowWrapper;
  beforeEach(() => wrapper = shallow(<ChorusCard buttonPressed={false}/>));

  it("should render correctly", () => expect(wrapper).toMatchSnapshot());

  it("should render a <div />", () => {
    expect(wrapper.find("div").length).toEqual(1);
  });
});
