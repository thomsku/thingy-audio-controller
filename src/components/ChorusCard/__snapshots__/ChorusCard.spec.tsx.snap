// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlatButton should render correctly 1`] = `ShallowWrapper {}`;
